src/game_api/level_api.cpp

```diff
@@ -1880,6 +1813,26 @@ uint32_t ThemeInfo::get_aux_id()
 void LevelGenSystem::init()
 {
     data->init();
+
+    for (auto theme : themes)
+    {
+        if (theme == theme_arena) // no reason to?
+            continue;
+
+        hook_vtable<void(ThemeInfo*), 0x15>( // spawn_transition
```

Comment:
So why does this work but not the thing in hook_themes?

Reply:
And even more curious why the other hooks there do work but not this...

Reply:
I tried to check but can't setup any debugging or export .pdb file (to use symbols in x64dbg) for release.
Was thinking maybe it's a thread issue, but changing `ptr` to `ptr_main` in get_entities doesn't solve this
So until you get to ask @Malacath-92 about this it remains mystery